bindings/java/c/evmc-vm.c

```diff
@@ -7,6 +7,12 @@
 #include "evmc/loader.h"
 #include "host.h"
 
+static void throw_java_assert(JNIEnv* jenv, const char* msg)
+{
+    jclass jclazz = (*jenv)->FindClass(jenv, "java/lang/AssertionError");
```

Comment:
```suggestion
    jclass jcls = (*jenv)->FindClass(jenv, "java/lang/AssertionError");
```

Generally using variable names as `jxxx` causes troubles because they clash if JNI defined type names. E.g. every usage of `jvalue` clashes with `jvalue` type.

Maybe name variables `j_class` or `jClass`. But I'm also fine with having file-scope consistent mapping: `jclass -> jcls`.

Reply:
Let me know a naming scheme and we can apply it to the file.

Reply:
`jcls`.

Reply:
Was hard to see on the phone :)